fs/fs.go

```diff
@@ -404,7 +404,13 @@ func (self *RealFsInfo) GetFsInfoForPath(mountSet map[string]struct{}) ([]Fs, er
 				err error
 				fs  Fs
 			)
-			switch partition.fsType {
+			var fsType = partition.fsType
+			if fsType == ZFS.String() {
+				if _, devZfs := os.Stat("/dev/zfs"); os.IsNotExist(devZfs) {
```

Comment:
I'm not a fan of defining a new fsType.  Maybe put this inside of `case ZFS.String()`, and use `goto default` to revert to the default behavior when this is the case?

Reply:
Thank you for the review @dashpole. I moved the `/dev/zfs` check inside the ZFS case. Are you firmly set on using goto with an extra label (I believe the case label default does not qualify as a goto target)? I went with `fallthrough` what do you think?

Reply:
oh, if you can't goto default, then I don't know if that will work...  Let me take another look